tests/test_behavior/test_internal_video_interface.py

```diff
@@ -0,0 +1,293 @@
+from copy import deepcopy
+from datetime import datetime
+
+import numpy as np
+import pytest
+from dateutil.tz import gettz
+from pynwb import NWBHDF5IO
+
+from neuroconv import NWBConverter
+from neuroconv.datainterfaces.behavior.video.internalvideointerface import (
+    InternalVideoInterface,
+)
+from neuroconv.utils import dict_deep_update
+
+
+@pytest.fixture
+def cv2():
```

Comment:
This is not how this is used:
See the documentation that I shared and other usees:

https://github.com/search?q=org%3Acatalystneuro%20%20importorskip&type=code

Reply:
My bad, should be fixed now.